shepherd_bms/src/analyzer.cpp

```diff
@@ -76,6 +77,11 @@ void Analyzer::calcPackTemps()
             }
 
             total_temp += bmsdata->chip_data[c].thermistor_value[therm];
+            total_seg_temp += bmsdata->chip_data[c].thermistor_value[therm];
+        }
+        if (c % 2 == 0) {
+            bmsdata->segment_average_temps[c/2] = total_seg_temp / 22;
```

Comment:
What is 22 in this case? Number of therms?

Reply:
indeed 1/4 of 88 which is whats used below